src/components/OrganisationEditor/index.tsx

```diff
@@ -1,8 +1,9 @@
-import React, { useState, FC, FormEvent, useEffect } from 'react';
-import { CreateOrganization, Organization } from '../../generated-api-client';
-import { Input } from '../../components/InputField';
-import { Button } from '../../components/Button';
 import _ from 'lodash';
+import React, { FC, FormEvent, useEffect, useState } from 'react';
```

Comment:
Hier kann dann "React" auch raus und in Zeile 36, wo es für einen useState benutzt wird